src/tactic/interactive_expr.lean

```diff
@@ -390,8 +393,16 @@ tc.stateless $ λ ft, do
   let lcs := group_local_collection lcs,
   lchs ← lcs.mmap (λ lc, do
     lh ← local_c lc,
-    let ns : list (html γ) := lc.locals.map $ λ n,
-      h "span" [cn "goal-hyp b pr2", key n.local_uniq_name] [html.of_name n.local_pp_name],
+    ns ← lc.locals.mmap (λ n, do -- extract into function?
+      cls ← is_class lc.type,
```

Comment:
This (and the `if ¬ cls`) can go outside the `mmap` loop

Reply:
I could move `is_class`, but the branches of the if-then-else depend on `n` so I'm not sure how that makes sense.

Reply:
Good point, I think the indents were confusing me